src/components/listing-search-page/SearchResults/listing-search-cards.style.js

```diff
@@ -37,14 +39,58 @@ export const SearchButton = styled.button`
 
 export const CardsContainer = styled.div`
   max-height: 40rem;
-  overflow: scroll;
+  overflow-y: scroll;
+`;
+
+export const SorryMessage = styled.div`
```

Comment:
```suggestion
export const Container = styled.div`
```
To hold both messages